detections/cloud/azure_active_directory_high_risk_sign_in.yml

```diff
@@ -14,10 +14,11 @@ description: The following analytic detects high-risk sign-in attempts against A
   exploitation within the environment.
 data_source:
 - Azure Active Directory
-search: '`azure_monitor_aad` category=UserRiskEvents properties.riskLevel=high | rename
-  properties.* as * | stats count min(_time) as firstTime max(_time) as lastTime values(user)
-  as user by src_ip, activity, riskLevel, riskEventType, additionalInfo | `security_content_ctime(firstTime)`
-  | `security_content_ctime(lastTime)` | `azure_active_directory_high_risk_sign_in_filter`'
+search: '`azure_monitor_aad` `azure_monitor_aad` category=UserRiskEvents properties.riskLevel=high
+  | rename category as action
```

Comment:
We should not rename category as action.

Reply:
fixed it